### ISSUE TITLE
Trying to access an eslint configuration that is not a dependency of react-error-boundary

### ISSUE THREAD

**Issue by mrbinky3000** [bvaughn/react-error-boundary#181]
- `react-error-boundary` version: 4.0.12
- `node` version: 20.10.0
- `npm` version: 10.2.3

Relevant code or config

```json
// package.json file for react-error-boundary
{
  "eslintConfig": {
    "extends": "./node_modules/kcd-scripts/eslint.js",
    "rules": {
      "react/prop-types": "off",
      "react/no-did-update-set-state": "off",
      "babel/no-unused-expressions": "off"
    }
  },
}
```

What you did:
Deleted my project's package-lock.json, ran `npm install` to get latest versions of things.  I am using a monorepo.  A package in my monorepo is using react-error-boundary.  I am not using it directly.

What happened:
I get the following error when I try to build my packages in my monorepo

```
    [!] (plugin commonjs--resolver) Error: Failed to load config "./node_modules/kcd-scripts/eslint.js" to extend from.
    Referenced from: /Users/mtoledo/Express/thats-so-raven/node_modules/react-error-boundary/package.json
    ../../node_modules/react-error-boundary/dist/react-error-boundary.esm.js
        at configInvalidError /Users/me/my-project/node_modules/@eslint/eslintrc/lib/config-array-factory.js:302:9)
        at ConfigArrayFactory._loadExtendedShareableConfig /Users/me/my-project/node_modules/@eslint/eslintrc/lib/config-array-factory.js:933:23)
        at ConfigArrayFactory._loadExtends /Users/me/my-project/node_modules/@eslint/eslintrc/lib/config-array-factory.js:810:25)
        at ConfigArrayFactory._normalizeObjectConfigDataBody /Users/me/my-project/node_modules/@eslint/eslintrc/lib/config-array-factory.js:749:25)
        at _normalizeObjectConfigDataBody.next (<anonymous>)
        at ConfigArrayFactory._normalizeObjectConfigData /Users/me/my-project/node_modules/@eslint/eslintrc/lib/config-array-factory.js:694:20)
        at _normalizeObjectConfigData.next (<anonymous>)
        at ConfigArrayFactory.loadInDirectory /Users/me/my-project/node_modules/@eslint/eslintrc/lib/config-array-factory.js:540:28)
        at CascadingConfigArrayFactory._loadConfigInAncestors /Users/me/my-project/node_modules/@eslint/eslintrc/lib/cascading-config-array-factory.js:392:46)
        at CascadingConfigArrayFactory._loadConfigInAncestors /Users/me/my-project/node_modules/@eslint/eslintrc/lib/cascading-config-array-factory.js:411:20)
```

Reproduction repository:

This would be difficult.

Problem description:

There seems to be a hard-coded path for an eslint config.  You are referencinc kcd-scripts  yet  kcd-scripts is not a dependency of your project.

Suggested solution:

Not sure.  Try and find a different way to access kcd-scripts?  Perhaps make it a dependency?

**Comment by mrbinky3000** [comment]
Removing the line `"extends": "./node_modules/kcd-scripts/eslint.js",` allowed my project to build.

**Comment by bvaughn** [comment]
Hmm, I'm confused. The `react-error-boundary/package.json` file does not contain any reference to `kcd-scripts`:
https://github.com/bvaughn/react-error-boundary/blob/master/package.json

I'm not sure how/where that got added in.

**Comment by bvaughn** [comment]
`kcd-scripts` is one of @kentcdodds's packages. I think he added some references so it (#76) but I thought I removed them when migrating to v4 (#131). I'm not sure how it ended up in newer releases honestly.

I just did a clean checkout+build of this project, and published it as version 4.0.13. There are no references to `kcd-scripts` in that release. I hope this fixes things for you.

**Comment by mrbinky3000** [comment]
That worked, thanks!
